webapi/_test_data/test_route/texts/beol/BEOLTEIMapping.xml

```diff
@@ -44,6 +44,13 @@
         </tag>
         <standoffClass>
             <classIri>http://www.knora.org/ontology/0801/beol#StandoffEntityTag</classIri>
+            <attributes>
```

Comment:
in the mapping of LEOO I have the same standoffEntityTag definition but with attribute type? won't this duplicate cause error?

`<standoffClass>
            <classIri>http://www.knora.org/ontology/0801/beol#StandoffEntityTag</classIri>
             <attributes>
                <attribute>
                    <attributeName>type</attributeName>
                    <namespace>noNamespace</namespace>
                    <propertyIri>http://www.knora.org/ontology/0801/beol#standoffEntityType</propertyIri>
                </attribute>
            </attributes>
            <datatype>
                <type>http://www.knora.org/ontology/knora-base#StandoffLinkTag</type>
                <attributeName>ref</attributeName>
            </datatype>
        </standoffClass>`

Reply:
I sorry, I am not sure I understand your question correctly: are you worried about a conflict because the same standoff tags is used in BEBB and LEOO mappings?

Reply:
I believe I don't have a complete understanding of how mappings are used for TEI export. I guess the mapping for TEI export is in disregard of the mappings used for import of the letters. is it so? or are you taking into account the mappings used for import in the process of export of TEI?

Reply:
> I guess the mapping for TEI export is in disregard of the mappings used for import of the letters.

Exactly. The TEI/XML mapping is a virtual mapping that creates TEI/XML from standoff, regardless of what the original XML looked like.